src/stories/UI/SlateArea.examples.tsx

```diff
@@ -0,0 +1,21 @@
+import SlateArea from "../../components/SlateArea/SlateArea";
+import { defaultFeatureSet } from "../../components/SlateArea/featureSets";
+
+export const BasicSlateArea = () => {
+  const mediaFunctionalityProps = {
+    featureSet: [...defaultFeatureSet],
+    // InlineMenu,
+    // inlineMenuProps: { triggerWidget }
+  };
+  return (
+    <SlateArea
```

Comment:
We can simplify this basic example to something like:
```ts
<SlateArea
        onFocus={() => console.log("onFocus")}
        onChange={(value) => console.log("onChange", value)}
        name="body"
        defaultValue="Hi"
      />
```

(The default feature set is added by default so we don't need to pass it in.)